src/common.jl

```diff
@@ -1,10 +1,18 @@
 ## sample space/domain
 
+"""
+`F <: VariateForm` specifies the form of the variate or
+dimension of a sample, univariate (scalar), multivariate (vector), matrix-variate (matrix).
```

Comment:
```suggestion
dimension of a sample: univariate (scalar), multivariate (vector) or matrix-variate (matrix).
```

Reply:
I didn't want to put it like an exhaustive list here, anyone can subtype `VariateForm` and define their own

Reply:
My bad then :)